css/css-contain/contain-layout-breaks-001.html

```diff
@@ -5,7 +5,7 @@
   <link rel="author" title="Florian Rivoal" href="https://florian.rivoal.net">
   <meta name=flags content="">
   <meta name=assert content="layout containment allows forced breaks.">
-  <link rel="match" href="reference/contain-style-breaks-004-ref.html">
+  <link rel="match" href="reference/contain-two-blocks-a-letters-two-lines.html">
   <link rel=help href="https://drafts.csswg.org/css-contain-1/#containment-layout">
```

Comment:
The current naming follows an established convention, documented in the csswg's wiki: https://wiki.csswg.org/test/reftest#the-reftest-reference-file

> References should be named after the earliest test that uses them in the test-topic series they belong to, and must have either -ref or -notref appended to the name
>[...]
> For example, if two self-describing tests list-style-type-003.xht and list-style-type-004.xht share the same reference, that reference could be named list-style-type-003-ref.xht. 

I don't have a strong opinion, but tons of existing tests references are named like that. However, that convention is not documented in https://web-platform-tests.org/ which doesn't have anything to say about reference naming at all, as far as I can tell.

So, should we preserve that convention (and if so, document it on web-platform-tests.org)? Should we replace it with something else (and document that)?

CCing @gsnedders @foolip @fantasai @svgeesus for opinion

Reply:
Ok, I didn't know about that convention.
The documentation in https://web-platform-tests.org/writing-tests/reftests.html says:
> For references that are applicable only to a single test, it is recommended to use the test name with a suffix of -ref as their filename; e.g., test.html would have test-ref.html as a reference.

But it doesn't mention what happens when more tests start to use it.

I believe @mstensho has a point that having a test `contain-layout-005` pointing to `contain-size-001-ref` is somehow confusing, but I don't have a strong opinion on this topic either.

I can left these changes out of the PR, and we can discuss in an issue if needed.
I believe the changes reusing the the references from `css/reference/` are welcomed by everyone, aren't they?

Reply:
Just an opinion: when there's a number in the ref file and a second test is added using the same ref, rename the ref to something more generic like the ref-filled-green-200px-square.html files in css/reference/

Reply:
I agree with the attributed-to-@mstensho view that pointing to other numbered refs is confusing.

On the whole, I think filename rules aren't that beneficial (in general, it's probably better to try and make the file name descriptive than adding an incrementing number onto it!).

Reply:
@frivoal are you planing to review this again? Or what should we do move this forward?

I can split this in 2 PRs:
* One for using generic references in the current tests when possible.
* Another one for changing the names, once there's agreement about that being the right thing todo.

Reply:
> * One for using generic references in the current tests when possible.

This has been done in #11667. Once that's merged I'd update this PR with the other part.